content/en/smart-contracts/fundamentals/the-filecoin-virtual-machine/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "The FVM"
-description: "The Filecoin virtual machine (FVM) is a runtime environment for smart contracts on the Filecoin network. These smart contracts, also called actors, can be written in Solidity."
-lead: "The Filecoin virtual machine (FVM) is a runtime environment for smart contracts on the Filecoin network. These smart contracts, also called actors, can be written in Solidity."
+description: "The Filecoin Virtual Machine (FVM) is a runtime environment enabling users to deploy their own smart contracts on the Filecoin blockchain.  These smart contracts are called “actors,” and allow for on-chain computation, or “computation over state,” on Filecoin."
```

Comment:
I would try to refrain from changing the substance of the article, since this has gone through many reviews and this is what the FVM DX team has settled on, but if you think this is more conducive to reader understanding, you can go ahead and commit the change.

Reply:
```suggestion
description: "The Filecoin Virtual Machine (FVM) is a runtime environment enabling users to deploy their own smart contracts on the Filecoin blockchain.  These smart contracts are called “actors,” and allow for on-chain computation, or “computation aieufhgaljehgkfajh over state,” on Filecoin."
```